src/router/index.ts

```diff
@@ -19,6 +19,31 @@ import BackendServiceEditor from '@/doc-editors/BackendServicesEditor.vue'
 import HelpAndSupport from '@/views/HelpAndSupport.vue'
 import DashboardDisplay from '@/views/Dashboards.vue'
 import DynamicRulesList from '@/views/DynamicRulesList.vue'
+import DynamicRulesEditor from '@/doc-editors/DynamicRulesEditor.vue'
+import EdgeFunctionsList from '@/views/EdgeFunctionsList.vue'
+import EdgeFunctionsEditor from '@/doc-editors/EdgeFunctionsEditor.vue'
+import PremiumPage from '@/views/PremiumPage.vue'
+import RequestsUtils from '@/assets/RequestsUtils'
+
+
+async function premiumServerIsLive(to: any, from: any) {
```

Comment:
Looks like we're not using `to` and `from` in this funciton
```suggestion
async function premiumServerIsLive() {
```